common/flows/src/main/scala/weco/flows/FlowOps.scala

```diff
@@ -51,4 +53,20 @@ trait FlowOps extends Logging {
           (ctx, result)
       }
       .collect { case (ctx, Right(data)) => (ctx, data) }
+
+  /**
+    * Broadcasts the output of a flow to flows `a` and `b` and merges them again
+    */
```

Comment:
It might be helpful to explain what "merge" means in this context – you get a new flow with all the outputs from both flows, in any order.

e.g. if `a` produced `(1, 2, 3, …)` and `b` produced `(-1, -2, -3, …)`, you might get `(1, -1, 2, 3, 4, -2, 5, -3)`.

Or have I misunderstood?

Reply:
yes, I will add it